examples/src/main/java/io/dapr/examples/actors/http/DemoActorClient.java

```diff
@@ -56,19 +56,19 @@ public static void main(String[] args) throws InterruptedException {
    * Makes multiple method calls into actor until interrupted.
    * @param actor Actor to be invoked.
    */
-  private static final void callActorForever(ActorProxy actor) {
+  private static final void callActorForever(String actorId, DemoActor actor) {
     // First, register reminder.
-    actor.invokeActorMethod("registerReminder").block();
+    actor.registerReminder();
 
     // Now, we run until thread is interrupted.
     while (!Thread.currentThread().isInterrupted()) {
       // Invoke actor method to increment counter by 1, then build message.
-      int messageNumber = actor.invokeActorMethod("incrementAndGet", 1, int.class).block();
-      String message = String.format("Actor %s said message #%d", actor.getActorId().toString(), messageNumber);
+      int messageNumber = actor.incrementAndGet(1).block();
+      String message = String.format("Actor %s said message #%d", actorId, messageNumber);
 
       // Invoke the 'say' method in actor.
-      String result = actor.invokeActorMethod("say", message, String.class).block();
-      System.out.println(String.format("Actor %s got a reply: %s", actor.getActorId().toString(), result));
+      String result = actor.say(message);

```

Comment:
I like it :)

Reply:
![image](https://user-images.githubusercontent.com/130954/75079710-8b8cc780-54be-11ea-884d-9b1cb73aac60.png)